lib/Composer.php

```diff
@@ -197,7 +197,7 @@ protected static function getOptions(Event $event): array
             $options['symfony-cache-warmup'] = $_SERVER['SYMFONY_CACHE_WARMUP'];
         }
 
-        $options['process-timeout'] = $event->getComposer()->getConfig()->get('process-timeout');
+        $options['process-timeout'] = (int) $event->getComposer()->getConfig()->get('process-timeout');
```

Comment:
Should we better leave it out completely if not explicitly set and fallback to the default instead? 🤔

Reply:
this was my first intention :D